ziti/cmd/create/config_templates/controller.yml

```diff
@@ -7,7 +7,7 @@ v: 3
 #  memory:
 #    path: ctrl.memprof
 
-db:                     "{{ .ZitiHome }}/db/ctrl.db"
+db:                     "{{ .ZitiHome }}/{{ .Controller.Database.DatabaseFile }}"
```

Comment:
use same default, but allow overriding with `ZITI_CTRL_DATABASE_FILE`.

Reply:
remove the `{{ .ZitiHome }}/` bit and add/expect it in the `.Controller.Database.DatabaseFile` field, pls

Reply:
In case it's an unintended consequence, this will change the way the environment is computed to include the full path:

Before:

```
ZITI_HOME=/path/to/cwd
ZITI_CTRL_DATABASE_FILE=db/ctrl.db
```

After:

```
ZITI_HOME=/path/to/cwd
ZITI_CTRL_DATABASE_FILE=/path/to/cwd/db/ctrl.db
```

It will also entail parsing any value set in `ZITI_CTRL_DATABASE_FILE` to determine whether or not it's a qualified path, and if not, to assume the qualified path is relative to `ZITI_HOME`.